src/pages/timeRestrictions.tsx

```diff
@@ -0,0 +1,109 @@
+                            <FormElementWrapper
+                                errors={errors}
+                                errorId={timeRestrictionsErrorId}
+                                errorClass="govuk-radios--errors"
+                            >
+                                <div className="govuk-radios">
+                                    <div className="govuk-radios__item">
+                                        <input
+                                            className={`govuk-radios__input ${
+                                                errors.length > 0 ? 'govuk-input--error' : ''
+                                            } `}
+                                            id="time-restrictions-yes"
+                                            name="timeRestrictions"
+                                            type="radio"
+                                            value="yes"
+                                        />
+                                        <label
+                                            className="govuk-label govuk-radios__label"
+                                            htmlFor="time-restrictions-yes"
+                                        >
+                                            Yes
+                                        </label>
+                                    </div>
+                                    <div className="govuk-radios__item">
+                                        <input
+                                            className={`govuk-radios__input ${
+                                                errors.length > 0 ? 'govuk-input--error' : ''
+                                            } `}
+                                            id="time-restrictions-no"
+                                            name="timeRestrictions"
+                                            type="radio"
+                                            value="no"
+                                        />
+                                        <label
+                                            className="govuk-label govuk-radios__label"
+                                            htmlFor="time-restrictions-no"
+                                        >
+                                            No
+                                        </label>
+                                    </div>
+                                </div>
+                            </FormElementWrapper>
+                        </fieldset>
+                    </div>
+                    <input type="submit" value="Continue" id="continue-button" className="govuk-button" />
+                </>
+            </CsrfForm>
+        </TwoThirdsLayout>
+    );
+};
+
+export const getServerSideProps = (ctx: NextPageContextWithSession): { props: TimeRestrictionsProps } => {
+    const timeRestrictionsInfo = getSessionAttribute(ctx.req, TIME_RESTRICTIONS_ATTRIBUTE);
```

Comment:
By removing the `TimeRestrictionsAttribute` from the API, this variable would then only ever be undefined or `TimeRestrictionsAttributeWithErrors`. Hence, we could have a very simple getServerSideProps which just sets an errors array 😄